docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -0,0 +1,728 @@
+your application already implements connection logic that tests for 
+liveness and readiness, you should use the SRV records of the Pods in the 
+StatefulSet (e.g `web-0.nginx.default.svc.cluster.local`, 
+`web-1.nginx.default.svc.cluster.local`).
+
+#### Stable Storage
+
+Get the PersistentVolumeClaims for `web-0` and `web-1`.
+
+```shell
+kubectl get pvc -l app=nginx
+NAME        STATUS    VOLUME                                     CAPACITY   ACCESSMODES   AGE
+www-web-0   Bound     pvc-15c268c7-b507-11e6-932f-42010a800002   1Gi        RWO           48s
+www-web-1   Bound     pvc-15c79307-b507-11e6-932f-42010a800002   1Gi        RWO           48s
+```
+The StatefulSet controller created two PersistentVolumeClaims that are 
+bound to two [PersistentVolumes](/docs/user-guide/volumes/). As the cluster used 
+in this tutorial is configured to dynamically provision PersistentVolumes, the 
+PersistentVolumes were created and bound automatically.
+
+The NGINX webservers, by default, will serve an index file at 
+`/usr/share/nginx/html/index.html`. The `volumeMounts` field in the 
+StatefulSets `spec` ensures that the `/usr/share/nginx/html` directory is 
+backed by a PersistentVolume.
+
+Write the Pods' hostnames to their `index.html` files and verify that the NGINX 
+webservers serve the hostnames.
+
+```shell
+for i in 0 1; do kubectl exec web-$i -- sh -c 'echo $(hostname) > /usr/share/nginx/html/index.html'; done
+
+for i in 0 1; do kubectl exec -it web-$i -- curl localhost; done
+web-0
+web-1
+```
+
+In one terminal, watch the StatefulSet's Pods.
+
+```shell
+kubectl get pod -w -l app=nginx
+```
+
+In a second terminal, delete all of the StatefulSet's Pods.
+
+```shell
+kubectl delete pod -l app=nginx
+pod "web-0" deleted
+pod "web-1" deleted
+```
+Examine the output of the `kubectl get` command in the first terminal, and wait 
+for all of the Pods to transition to Running and Ready.
+
+```shell
+kubectl get pod -w -l app=nginx
+NAME      READY     STATUS              RESTARTS   AGE
+web-0     0/1       ContainerCreating   0          0s
+NAME      READY     STATUS    RESTARTS   AGE
+web-0     1/1       Running   0          2s
+web-1     0/1       Pending   0         0s
+web-1     0/1       Pending   0         0s
+web-1     0/1       ContainerCreating   0         0s
+web-1     1/1       Running   0         34s
+```
+
+Verify the web servers continue to serve their hostnames.
+
+```
+for i in 0 1; do kubectl exec -it web-$i -- curl localhost; done
+web-0
+web-1
+```
+
+Event though `web-0` and `web-1` were rescheduled, they continue to serve their 
+hostnames because the PersistentVolumes associated with their Persistent 
+Volume Claims are remounted to their `volumeMount`s. No matter what node `web-0`
+and `web-1` are scheduled on, their PersistentVolumes will be mounted to the 
+appropriate mount points.
+
+### Scaling a StatefulSet
+When we refer to scaling a StatefulSet, we mean increasing or decreasing the 
```

Comment:
Avoid "we" constructs. Address the user directly. If you're defining terms, just define the terms without using "we."

For example: "Scaling a StatefulSet means increasing or decreasing the number of replicas in the StatefulSet. You scale a StatefulSet by updating the `replicas` field."